chia/wallet/util/debug_spend_bundle.py

```diff
@@ -160,7 +161,7 @@ def debug_spend_bundle(spend_bundle, agg_sig_additional_data=DEFAULT_CONSTANTS.A
     created = set(spend_bundle.additions())
     spent = set(spend_bundle.removals())
 
-    zero_coin_set = {coin.name() for coin in created if coin.amount == 0}
+    zero_coin_set = {bytes32(coin.name()) for coin in created if coin.amount == 0}
```

Comment:
`Coin.name()` is supposed to return a `bytes32` already.. is it not?

Reply:
Strangely the `Coin.name()` returned `bytes` during my testing despite being imported from chia_rs / untouched by me